common/src/main/java/com/revenuecat/purchases/common/Dispatcher.kt

```diff
@@ -41,6 +41,14 @@ open class Dispatcher(
         }
     }
 
+    fun executeOnBackground(command: () -> Unit) {
```

Comment:
I'm a bit confused about the difference between this and `enqueue`...
this method doesn't guarantee that the code runs in the background, right? It just runs on whatever the executorService was set to? 

it seems like the biggest difference with them is just that they take a different param type? should it be an overload of `enqueue` instead? 
Also, should the check for `!executorService.isShutdown` also happen on `enqueue`?

Reply:
You are totally right. I made it an overload with Runnable initially and then moved the one from Purchases and forgot to change the name.

Regarding enqueue checking for isShutdown... I guess it can't hurt to add right? Otherwise, the app crashes